src/pages/home/report/ReportActionItem.tsx

```diff
@@ -699,9 +695,8 @@ function ReportActionItem({
         if (ReportUtils.isTaskReport(report)) {
             if (ReportUtils.isCanceledTaskReport(report, parentReportAction)) {
                 return (
-                    <View style={[StyleUtils.getReportWelcomeContainerStyle(isSmallScreenWidth)]}>
-                        <AnimatedEmptyStateBackground />
-                        <View style={[StyleUtils.getReportWelcomeTopMarginStyle(isSmallScreenWidth)]}>
+                    <View>
+                        <View>
```

Comment:
Does a single prop-less `View` make any sense in React Native? Leave alone two.

Reply:
@rayane-djouah Does a _single_ prop-less `View` make any difference? I could imagine that in _some_ cases it might make a difference on Web, but I would still test it.

Unless you know or strongly suspect that it **is** needed, than please explain it to me.

Reply:
In most cases, a fragment `<>` will behave the same, while it is a simpler concept than a `View`.

Reply:
(of course, when multiple children are involved)